dataflux_pytorch/benchmark/checkpointing/multinode/train.py

```diff
@@ -53,6 +55,9 @@ def validate(args):
 
 def get_strategy(args, project, model, ckpt_dir_path):
     strategy = None
+    policy = {
```

Comment:
Just to clarify, does this makes it so that FSDP only applies to the specified layers, and other layers are not managed by the FSDP strategy?

Reply:
Right, other top-level layers in the model will be duplicated across all copies. Since these are the only large ones it performs as expected.